sklearn/discriminant_analysis.py

```diff
@@ -622,9 +622,9 @@ def __init__(self, priors=None, reg_param=0., store_covariance=False,
         self.tol = tol
 
     @property
-    @deprecated("Attribute covariances_ was deprecated in version"
+    @deprecated("Attribute ``covariances_`` was deprecated in version"
```

Comment:
Test are broken because of the changes :)

Reply:
trying to fix it